translator/writer/writer.h

```diff
@@ -49,11 +49,14 @@ class UsdArnoldWriter {
     void setWriteBuiltin(bool b) { _writeBuiltin = b;}
     bool getWriteBuiltin() const { return _writeBuiltin;}
 
+    bool isNodeExported(const std::string &name) { return _exportedNodes.find(name) != _exportedNodes.end();}
```

Comment:
We could use `std::unordered_map::count` here: `_exportedNodes.count(name) == 1`

[See reference](https://en.cppreference.com/w/cpp/container/unordered_map/count)

> Returns the number of elements with key that compares equal to the specified argument key, which is either 1 or 0 since this container does not allow duplicates.